lib/tmuxinator/config.rb

```diff
@@ -56,6 +60,12 @@ def default
         "#{directory}/default.yml"
       end
 
+      def options
+        YAML.safe_load(File.read(options_file), [], [], true) || {}
```

Comment:
@flovilmart Would you mind using the new safe_load method signature (sans positional parameters two, three and four)? As-is, this will cause errors in some environments because of changes to Psych and libyaml.

It would look something like this: 

`YAML.safe_load(File.read(options_file), aliases: true)`

See [this](https://github.com/tmuxinator/tmuxinator/pull/823/files) PR for more context.